graylog2-server/src/main/java/org/graylog2/commands/Token/AbstractAutomationTokenCommand.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog2.commands.Token;
+
+import com.github.joschi.jadconfig.Parameter;
+import com.google.inject.Module;
+import jakarta.annotation.Nonnull;
+import java.util.List;
+import org.graylog2.Configuration;
+import org.graylog2.bindings.ConfigurationModule;
+import org.graylog2.cluster.ClusterConfigServiceImpl;
+import org.graylog2.commands.AbstractNodeCommand;
+import org.graylog2.featureflag.FeatureFlags;
+import org.graylog2.plugin.cluster.ClusterConfigService;
+
+public abstract class AbstractAutomationTokenCommand extends AbstractNodeCommand {
```

Comment:
Can we get rid of this intermediate abstract class and merge the code with the actual implementation?

Reply:
I don't think it's going to be a problem, actually. But I don't want to block this PR for it.